pkg/cmd/admin/policy/reconcile_clusterroles.go

```diff
@@ -54,33 +60,22 @@ func NewCmdReconcileClusterRoles(name, fullName string, f *clientcmd.Factory, ou
 		Long:    reconcileLong,
 		Example: fmt.Sprintf(reconcileExample, fullName),
 		Run: func(cmd *cobra.Command, args []string) {
-			if err := o.Complete(cmd, f, args); err != nil {
-				kcmdutil.CheckErr(kcmdutil.UsageError(cmd, err.Error()))
-			}
-
-			changedClusterRoles, err := o.ChangedClusterRoles()
-			kcmdutil.CheckErr(err)
-
-			if len(changedClusterRoles) == 0 {
-				return
+			if err := o.Complete(cmd, f); err != nil {
+				kcmdutil.CheckErr(err)
 			}
 
-			if (len(kcmdutil.GetFlagString(cmd, "output")) != 0) && !o.Confirmed {
-				list := &kapi.List{}
-				for _, item := range changedClusterRoles {
-					list.Items = append(list.Items, item)
-				}
-
-				kcmdutil.CheckErr(f.Factory.PrintObject(cmd, list, out))
+			if err := o.Validate(args); err != nil {
+				kcmdutil.CheckErr(kcmdutil.UsageError(cmd, err.Error()))
 			}
 
-			if o.Confirmed {
-				kcmdutil.CheckErr(o.ReplaceChangedRoles(changedClusterRoles))
+			if err := o.RunReconcileClusterRoles(cmd, f); err != nil {
+				kcmdutil.CheckErr(err)
 			}
 		},
 	}
 
 	cmd.Flags().BoolVar(&o.Confirmed, "confirm", o.Confirmed, "Specify that cluster roles should be modified. Defaults to false, displaying what would be replaced but not actually replacing anything.")
+	cmd.Flags().BoolVar(&o.Union, "union-of", o.Union, "Get the union of the default and modified cluster roles.")
```

Comment:
I've never had a customer get on the phone and voluntarily use the words "union" or "intersection" in this context.  We need a better flag name.  `--additive-only`?  "Only adds policy rules to system roles, never removes them"